tests/by-util/test_tail.rs

```diff
@@ -94,15 +92,15 @@ fn test_stdin_redirect_file() {
         .run_no_wait();
 
     sleep(Duration::from_millis(500));
-    p.kill().unwrap();
+    let _ = p.kill();
```

Comment:
Is this a fix for something? I'd prefer unwrap, because it fails explicitly.

Reply:
Yes, this change made the test work on windows and maybe other platforms, I don't remeber currently. The cause may be, that it's an error on windows to try to kill a non running or already stopped process in contrast to unix. However, if the child can be killed successfully or not is not essential for this test. The `p.kill()` at this point only cleanups the test and the `_` indicates the `don't care for the result`. I don't want to be nitpicky but in tests I would actually prefer using an assertion, if I wanted or needed to be explicit about a result.

Reply:
This is super useful information, please add a comment to the line so we don't regress.

Reply:
@Joining7943 this is still unresolved :)